### PR TITLE
pie@v1.13.0 - SCSS linting GH action

### PR DESCRIPTION
**Pull request by jamieomaguire**
v1.13.0
------------------------------
*September 23, 2022*

### Added
- Linting step to GitHub workflow

### Changed
- Split out GitHub actions workflow into separate jobs

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -3,20 +3,77 @@ on:
...
+      - name: Cache Node Modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: node-modules-${{ hashFiles('**/yarn.lock') }}
+      # Run 'yarn' if cache doesn't exist. Use --prefer-offline to download packages from yarn cache folder where possible
+      - name: Install Dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: yarn add --cached
```
Comment (DamianMullins) [inline .github/workflows/build.yml:31]: Should this be `yarn install`?

Comment (jamieomaguire) [reply .github/workflows/build.yml:31]: I think so - the instructions were a bit weird and specified yarn add. This PR isn't quite ready for review though haha :D

File: apps/pie-microsite/src/assets/styles/base/_layout.scss [hunk 2]
```diff
@@ -4,16 +4,10 @@
     min-height: 100%;
     display: grid;
     grid-template: 64px 1fr max-content / 260px 1fr;
-    grid-template-areas:
-            'header header'
-            'nav main'
-            'nav footer';
+    grid-template-areas: 'header header' 'nav main' 'nav footer';
```
Comment (kevinrodrigues) [inline apps/pie-microsite/src/assets/styles/base/_layout.scss:7]: Why has this changed back?

File: apps/pie-microsite/src/assets/styles/base/_layout.scss [hunk 3]
```diff
@@ -4,16 +4,10 @@
...
 
     @include f.media('<=narrowMid') {
-        grid-template-areas:
-                'header header'
-                'main main'
-                'footer footer';
+        grid-template-areas: 'header header' 'main main' 'footer footer';
```
Comment (kevinrodrigues) [inline apps/pie-microsite/src/assets/styles/base/_layout.scss:10]: Same.

File: apps/pie-microsite/src/assets/styles/base/_layout.scss (hunk 2 above)
Comment (jamieomaguire) [reply apps/pie-microsite/src/assets/styles/base/_layout.scss:7]: linting said it had to be! how did you avoid it?

**Review by siggerzz — APPROVED** [review]
GH Actions look good 👍🏼

File: apps/pie-microsite/src/assets/styles/base/_layout.scss (hunk 2 above)
Comment (kevinrodrigues) [reply apps/pie-microsite/src/assets/styles/base/_layout.scss:7]: Was this a whilst pre-commiting? If so can you delete node modules and re-install. I added a fix to our config which should have pulled through from version 3.2.0 https://github.com/justeat/stylelint-config-fozzie/releases/tag/v3.2.0

Comment (jamieomaguire) [reply apps/pie-microsite/src/assets/styles/base/_layout.scss]: resolved the issue together offline - closing this thread :)

**Review by siggerzz — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**